server/router.go

```diff
@@ -6,9 +6,13 @@ import (
 	"github.com/uc-cdis/cohort-middleware/middlewares"
 	"github.com/uc-cdis/cohort-middleware/models"
 	gintrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/gin-gonic/gin"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
 func NewRouter() *gin.Engine {
+	tracer.Start()
+	defer tracer.Stop()
```

Comment:
should the tracer stay on for as long as we're serving http requests? If yes, then maybe the .Stop() should not be here but in `server/server.go`?